### PR TITLE
Add GPU notes to README

### PR DESCRIPTION
**Pull request by nkreeger**
Here are the proposed changes for when we ship GPU builds (probably tomorrow).

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-node/91)

### CODE REVIEW

**Comment by dsmilkov** [comment]
:lgtm_strong: Sweet! 

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---

*[README.md, line 10 at r1](https://reviewable.io/reviews/tensorflow/tfjs-node/91#-LD43Aqb88mY-Xd-BEaT:-LD43Aqb88mY-Xd-BEaU:b6bs0p4) ([raw file](https://github.com/tensorflow/tfjs-node/blob/91ad0cb53311e66b79a0a97e3300e3d05fb6f587/README.md#L10)):*
> ```Markdown
> TensorFlow.js for Node currently supports the following platforms:
> - Mac OS X 10.12.6 (Sierra) or higher
> - Linux CPU (Ubuntu 16.04 or higher)
> ```

curious, does it support Debian or other distros that we can add to this list? 

---

*[README.md, line 11 at r1](https://reviewable.io/reviews/tensorflow/tfjs-node/91#-LD43Ss4CGCB-kIhkfnZ:-LD43Ss4CGCB-kIhkfn_:bhhkcic) ([raw file](https://github.com/tensorflow/tfjs-node/blob/91ad0cb53311e66b79a0a97e3300e3d05fb6f587/README.md#L11)):*
> ```Markdown
> - Mac OS X 10.12.6 (Sierra) or higher
> - Linux CPU (Ubuntu 16.04 or higher)
> - Linux GPU (Cuda 9.0 w/ CUDNN v7) ([see installation instructions](https://www.tensorflow.org/install/install_linux))
> ```

is this again Ubuntu only? Good to be consistent with the previous bullet point.

---


*Comments from [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-node/91#-:-LD43Ytc1U96dXv9W08a:b7ux6sc)*

**Comment by camsjams** [comment]
Is it safe to assume that some of the Docker images will work as well (https://hub.docker.com/r/tensorflow/tensorflow/tags/)

**Comment by nkreeger** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions, some commit checks pending.

---

*[README.md, line 10 at r1](https://reviewable.io/reviews/tensorflow/tfjs-node/91#-LD43Aqb88mY-Xd-BEaT:-LD4TGf259_6KXM8wp-m:bxhpsta) ([raw file](https://github.com/tensorflow/tfjs-node/blob/91ad0cb53311e66b79a0a97e3300e3d05fb6f587/README.md#L10)):*
<details><summary><i>Previously, dsmilkov (Daniel Smilkov) wrote…</i></summary><blockquote>

curious, does it support Debian or other distros that we can add to this list? 
</blockquote></details>

Yeah it does - I'll update to reflect. FWIW the tensorflow.org documentation only references Ubuntu.

---

*[README.md, line 11 at r1](https://reviewable.io/reviews/tensorflow/tfjs-node/91#-LD43Ss4CGCB-kIhkfnZ:-LD4Vh-B0Qe0501-rdV-:b-896fix) ([raw file](https://github.com/tensorflow/tfjs-node/blob/91ad0cb53311e66b79a0a97e3300e3d05fb6f587/README.md#L11)):*
<details><summary><i>Previously, dsmilkov (Daniel Smilkov) wrote…</i></summary><blockquote>

is this again Ubuntu only? Good to be consistent with the previous bullet point.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-node/91)*

**Comment by nkreeger** [comment]
@camsjams Most likely yes, we're just matching our requirements to core TensorFlow: https://www.tensorflow.org/install/install_linux

Ubuntu 16.04 is the base requirement because chasing CUDA and other build requirements of core TF requires a more recent kernel + glib.

File: README.md [hunk 1]
```diff
@@ -3,20 +3,42 @@
...
 
-## Installing 
+## Installing
+
+TensorFlow.js for Node currently supports the following platforms:
+- Mac OS X 10.12.6 (Siera) or higher
+- Linux CPU (Ubuntu 16.04 or higher)
+- Linux GPU (Ubuntu 16.04 or higher and Cuda 9.0 w/ CUDNN v7) ([see installation instructions](https://www.tensorflow.org/install/install_linux))
+
+*Other Linux variants might also work but this project matches [core TensorFlow installation requirments](https://www.tensorflow.org/install/install_linux).*
```
Comment (ManrajGrover) [inline README.md]: Nit: `requirments` -> `requirements`

**Pull request merged**